lib/oli_web/components/delivery/scored_activities/scored_activities.ex

```diff
@@ -135,6 +136,18 @@ defmodule OliWeb.Components.Delivery.ScoredActivities do
   def render(assigns) do
     ~H"""
     <div>
+      <%= unless is_nil(@current_assessment) do %>
+        <button class="whitespace-nowrap" phx-click="back" phx-target={@myself}>
```

Comment:
What do you think about  ```<button :if={!is_nil(@current_assessment)} ...>...</button>``` ?

To be honest, I would love if we could just do 

```<button :unless={is_nil(@current_assessment)} ...>...</button>```

but I think that [HEEX special attribute](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#sigil_H/2-heex-extension-special-attributes) is not available yet

Reply:
I like your proposal. I often forget to use the if statement inside the tag!

Reply:
Yeah.. the `unless` definitely sounds appealing. @nicocirio you could propose that in the liveview repo.

I'm wondering if we could use something like

```
<button :if{@current_assessment not in [nil]}
```

That way my  brain doesn't need to flip the value when I reach the `!`

Also this other way

```
<button :if{@current_assessment != nil}
```

but I like the `not in` because it reads as a sentence.